docs/source/_templates/autosummary/model_class.rst

```diff
@@ -1,7 +1,58 @@
 ..
   autogenerated from source/_templates/autosummary/model_class.rst
 
+.. currentmodule:: torchaudio.models
+
+..
+
 {%- set methods=["forward"] %}
+{%- set helpers={
+        "torchaudio.models.RNNTBeamSearch": [
+            "Hypothesis",
+        ],
+    }
+-%}
+{%- set factory={
+        "torchaudio.models.ConvTasNet": [
```

Comment:
can remove the `See also: conv_tasnet_base` on this [page](https://output.circle-artifacts.com/output/job/555214bd-54dd-4905-ba15-4b83ce4b4153/artifacts/0/docs/generated/torchaudio.models.ConvTasNet.html#factory-functions)